api/GPUCanvasContext.json

```diff
@@ -14,9 +14,13 @@
           },
           "edge": "mirror",
           "firefox": {
+            "version_added": "preview",
+            "partial_implementation": true,
+            "notes": "Currently supported on Linux and Windows only. See <a href='https://bugzil.la/1616739'>bug 1616739</a>."
+          },
+          "firefox_android": {
```

Comment:
@Elchi3  item `canvas` below is in spec, but I can't see it in IDL. Am I missing something obvious in https://searchfox.org/mozilla-central/source/dom/webidl/WebGPU.webidl#1087

Reply:
Ignore that. I think my brain broke.